kratos/utilities/intersection_utilities.h

```diff
@@ -86,7 +84,32 @@ class KRATOS_API(KRATOS_CORE) IntersectionUtilities
     ///@{
 
     /**
-     * Find the 3D intersection of a line (bounded) with a triangle (bounded)
+     * @brief Find the 3D intersection of a line (bounded) with a triangle (bounded)
+     * @param rTrianglePoint1 Coordinates of the first point of the intersecting triangle
+     * @param rTrianglePoint2 Coordinates of the second point of the intersecting triangle
+     * @param rTrianglePoint3 Coordinates of the third point of the intersecting triangle
+     * @param rLinePoint1 Coordinates of the first point of the intersecting line
+     * @param rLinePoint2 Coordinates of the second point of the intersecting line
+     * @param rIntersectionPoint The intersection point coordinates
+     * @param Epsilon The tolerance considered
+     * @return The intersection type index:
+     * -1 (the triangle is degenerate)
+     * 0 (disjoint - no intersection)
+     * 1 (intersect in a unique point)
+     * 2 (are in the same plane)
+     */
+    static int ComputeTriangleLineIntersection(
+        const array_1d<double,3>& rTrianglePoint1,
+        const array_1d<double,3>& rTrianglePoint2,
+        const array_1d<double,3>& rTrianglePoint3,
+        const array_1d<double,3>& rLinePoint1,
+        const array_1d<double,3>& rLinePoint2,
+        array_1d<double,3>& rIntersectionPoint,
+        const double Epsilon = 1e-12
+        );
```

Comment:
```suggestion
        const double Epsilon = 1e-12);
```

Reply:
The former definition was already in that style